lib/trento/application/usecases/clusters/adapters/gen.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Trento.Clusters.Gen do
+  @moduledoc """
+  Request checks execution adapter
```

Comment:
Honestly, I find hard to understand the difference.
I find it as an adapter, because it adapts certain functionalities. In this case for Wanda and old runner.
I would say adapter and behaviour are used for the same thing

Reply:
> adapter and behaviour are used for the same thing

probably here's where the misunderstanding comes from :sweat_smile:
I thought an `adapter` being a concretization of an abstraction, the `behaviour`, not also the abstraction itself.

Looking at [Ecto](https://github.com/elixir-ecto/ecto/tree/master/lib/ecto/adapter) [repo](https://github.com/elixir-ecto/ecto/blob/master/lib/ecto/adapter.ex) clarifies that both abstraction and implementations might be referred to both as adapter.

:rocket: